### PR TITLE
Slither Integration doc

### PR DESCRIPTION
**Pull request by Aniket-Engg**
Instructions to use https://github.com/ethereum/remix-project/pull/1267

### CODE REVIEW

**Review by yann300 — APPROVED** [review]

File: docs/slither.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+The result of the analysis will be shown in the Remix IDE terminal 
+
+![](images/a-slither-analysis-success-terminal.png)
+
+and also in the **remixd** console.
+
+![](images/a-slither-analysis-success-remixd.png)
+
+If one want to run the analysis only for Slither, deselect `Select all` checkbox and click on `Run`. Now it will show only the Slither Analysis report.
```
Comment (ryestew) [inline docs/slither.md:54]: If one only wants to run the analysis with Slither, then ...

or 

To only run Slither Analysis, deselect ...

File: docs/slither.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+2. **Solc:** [https://docs.soliditylang.org/en/latest/installing-solidity.html](https://docs.soliditylang.org/en/latest/installing-solidity.html)
+3. **Solc-select:** [https://github.com/crytic/solc-select#quickstart](https://github.com/crytic/solc-select#quickstart)
+
+### How to use
+
+If a project is shared through remixd and `localhost` workspace is loaded in Remix IDE, there will be an extra checkbox shown in `Solidity Static Analysis` plugin with the label `Enable Slither Analysis`.
+
+![](images/a-slither-analysis.png)
+
+One can check this box to run the analysis using Slither along with the Remix.
```
Comment (ryestew) [inline docs/slither.md:38]: One can check this box to run the analysis using Slither along with Remix's analysis library.
or
Check this box to run Sither along with whichever Remix's analysis module are checked.

File: docs/slither.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+If one want to run the analysis only for Slither, deselect `Select all` checkbox and click on `Run`. Now it will show only the Slither Analysis report.
+
+![](images/a-slither-analysis-only.png)
+
+### More Details
+
+Analysis for Slither is run using the version set in `Solidity Compiler` plugin on Remix IDE. Slither is a CLI tool and requires `solc` to run the analysis. Before running the analysis, Slither websocket plugin checks if current version of solc is same as the version set in Remix IDE.
+
+Incase, compiler version set in Solidity Compiler is different from current version of solc installed locally, Slither websocket plugin will update the `solc` version same as required using `solc-select`.
```
Comment (ryestew) [inline docs/slither.md:62]: If the compiler version set in Solidity Compiler is different from current version of solc installed locally, the Slither websocket plugin will update the `solc` version to be the same as the required version using `solc-select`.

**Review by ryestew — CHANGES_REQUESTED** [review]

File: docs/slither.md (hunk 1 above)
Comment (Aniket-Engg) [reply docs/slither.md:54]: updated

File: docs/slither.md (hunk 2 above)
Comment (Aniket-Engg) [reply docs/slither.md:38]: updated

File: docs/slither.md (hunk 3 above)
Comment (Aniket-Engg) [reply docs/slither.md]: updated

**Review by ryestew — APPROVED** [review]

**Pull request merged**